src/tup/version.h

```diff
@@ -22,6 +22,6 @@
 #define tup_version_h
 
 /* This constant is defined by an automatically generated file */
-extern const char *tup_version;
```

Comment:
This was exactly what I did in the first version, but Mike preferred the variant with array. The effect is identical anyway (;